FlashSpace/Settings/WorkspacesSettingsView.swift

```diff
@@ -30,6 +31,40 @@ struct WorkspacesSettingsView: View {
                 .foregroundStyle(.secondary)
                 .font(.callout)
             }
+
+            Section(
+                header:
+                HStack {
+                    Text("Floating Apps")
+                    Spacer()
+                    Button(action: viewModel.addFloatingApp) {
+                        Image(systemName: "plus")
+                    }
+                },
+                footer:
+                Text("Floating applications remain visible across all workspaces.")
+                    .foregroundStyle(.secondary)
+                    .font(.callout)
+            ) {
+                VStack(alignment: .leading) {
+                    List(
+                        settings.floatingApps ?? [],
+                        id: \.self
+                    ) { app in
+                        HStack {
+                            Text(app)
+                            Spacer()
+                            Button {
+                                viewModel.deleteFloatingApp(app: app)
+                            } label: {
+                                Image(systemName: "x.circle.fill")
+                            }
```

Comment:
I experimented with some styling and it seems like this one looks best:

```swift
VStack(alignment: .leading) {
    ForEach(
        settings.floatingApps ?? [],
        id: \.self
    ) { app in
        HStack {
            Button {
                viewModel.deleteFloatingApp(app: app)
            } label: {
                Image(systemName: "x.circle.fill").opacity(0.8)
            }.buttonStyle(.borderless)
            Text(app)
        }
    }
}
```

Putting the button on the left side makes it easier to find the one corresponding with the app name:

<img width="557" alt="image" src="https://github.com/user-attachments/assets/00b8ef36-e008-4a39-a3fd-4cb4ae4656c3" />

Reply:
Yes, it looks good.